### DISCUSSION TITLE
Cannot download from https://eduitfree.com/

### DISCUSSION THREAD

**Discussion by lumalisan** [aclap-dev/video-downloadhelper#470] [Help and Q&A] [answered]
## Trying to download from

https://eduitfree.com/

## Specific video

https://eduitfree.com/courses/three-js-journey-the-ultimate-three-js-course

## Error
SideDownload error:
[out#0/mp4 @ 0000029e22ed6ec0] Output file does not contain any stream
Error opening output file G:\ThreeJs Journey\Three.js Journey - The ultimate Three.js course.mp4.
Error opening output files: Invalid argument

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-ES
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Lisier",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment, marked as answer]
Doesn't look like VDH will work on that site, even the beta extension fails. ffmpeg on it's own also fails with it.
Whether there is a fix or not I don't know since I'm a user like you. It worked in another tool called **yt-dlp**

**Reply by lumalisan** [reply]
Wow, it worked like a charm. Thank you very much! 😄

**Reply by baixiaoji** [reply]
the yt-dlp work? I am running error when  Extracting information
<img width="672" alt="image" src="https://github.com/user-attachments/assets/84e5c740-feff-4a99-b8ba-b12a986edc4f">

**Reply by Mattmjs** [reply]
@baixiaoji , the error means it doesn't find the streaming information. But when the url for the streaming information is found,
yt-dlp to runs into a roadblock with a 403 error. So they don't want anyone copying that video. Just find a way to screen record it.

**Reply by baixiaoji** [reply]
@Mattmjs   Thx for replay! so it's only the screen record way?

**Reply by Mattmjs** [reply]
@baixiaoji  Yes I only see it that way for your video.

**Comment by dadesigns41** [comment]
What happened to eduitfree? Is it no longer a thing?
